practice.css

```diff
@@ -148,8 +148,8 @@ body {
 }
 
 [data-hash$="*"] { overflow: hidden }
-[data-hash$="*"] > .full-out:not(:target) { visibility: hidden }
-[data-hash$="*"] .full-out:target .full-in { min-height: 100vh }
+[data-hash$="*"] > .full-out:not(:target) + .full-in { visibility: hidden }
+[data-hash$="*"] .full-out:target + .full-in { min-height: 100vh }
```

Comment:
brittler because now depends on markup order and gets complex to add addl features in same way like `#47+`  `#47-`